mapper/tedge_mapper/src/mapper.rs

```diff
@@ -19,6 +20,10 @@ pub fn make_valid_topic_or_panic(topic_name: &str) -> Topic {
     Topic::new(topic_name).expect("This is a valid topic name")
 }
 
+pub fn make_valid_topic_filter_or_panic(filter_name: &str) -> TopicFilter {
+    TopicFilter::new(filter_name).expect("This is a valid topic filter name")
```

Comment:
```suggestion
    TopicFilter::new(filter_name).expect("This is an invalid topic filter name")
```

Reply:
Good catch. The original function's expect message should be changed as well.